programs/github-pull-request/index.js

```diff
@@ -26,13 +40,18 @@ module.exports = async({token, user, repo, pr, bundle, file}) => {
 		throw new Error('Pull-request entity is not available. I have nowhere to comment my findings ☹️');
 	}
 
+	const githubAppUserToken = await getGithubAppUserToken({
```

Comment:
I think it's a bit missleading, always creating that instance. where in fact it wont be used.

I rather see on top level ->

`const apiToken = token || new await getGithubAppUserToken({ .... })`

Reply:
Not sure i followed, Im just saying, Once u always create that object, it seems ur going to use it, but in fact the undercode does `if (!appId || !appPrivateKey) { return; }` whichis confusing imo